docs/src/main/sphinx/functions/math.md

```diff
@@ -226,12 +226,34 @@ Returns the value of `string` interpreted as a base-`radix` number.
 Returns the base-`radix` representation of `x`.
 :::
 
+## Geometrical functions
+
+:::{function} cosine_distance(array(double), array(double)) -> double
+Calculates the cosine distance of two double arrays:
+
+```sql
+SELECT cosine_distance(ARRAY[1.0, 2.0], ARRAY[3.0, 4.0]);
+-- 0.01613008990009257
+```
+:::
+
+:::{function} cosine_similarity(array(double), array(double)) -> double
+Calculates the cosine similarity of two double arrays:
+
+```sql
+SELECT cosine_similarity(ARRAY[1.0, 2.0], ARRAY[3.0, 4.0]);
+-- 0.9838699100999074
+```
+:::
+
 ## Statistical functions
 
+(statistical-cosine-similarity)=
```

Comment:
It’s not a statistical function. It was in the wrong section in the first place

Reply:
The sparse version one is not statistical either?

Reply:
If not .. then I can maybe collapse the two into one and say x and y can be either arrays of double or sparse vectors and have two examples?

Reply:
No, they are the same function, over different representation of vectors

Reply:
I think adjusted appropriately now.